src/test/common/fdc3.user-channels.ts

```diff
@@ -148,10 +147,10 @@ export function createUserChannelTests(cc: ChannelControl<any,any>, documentatio
         const errorMessage = `\r\nSteps to reproduce:\r\n- App A adds context listener of type fdc3.instrument\r\n- App A joins channel 1\r\n- App A unsubscribes the listener\r\n- App B joins channel 1\r\n- App B broadcasts context of type fdc3.instrument${documentation}`;
 
         const resolveExecutionCompleteListener = cc.initCompleteListener(scTestId7)
-        cc.setupAndValidateListener1(null, "unexpected-context", errorMessage, () =>  {/* noop */});
+        await cc.setupAndValidateListener1(null, "unexpected-context", errorMessage, () =>  {/* noop */});
```

Comment:
see previous comments about non-ideal failure message. This is used often enough (checking that context isNOT received) that it should be improved with its own util function (fail on receipt of context).